docs/language/api.rst

```diff
@@ -415,13 +415,14 @@ They can be used to assign multiple variables at once:
 defclass
 --------
 
-New classes are declared with ``defclass``. It can takes two optional parameters:
-a vector defining a possible super classes and another vector containing
+New classes are declared with ``defclass``. It can takes three optional parameters in the following order:
+a vector defining a possible super classes, a string (:term:`py:docstring`) and another vector containing
```

Comment:
`a vector defining a possible super classes` should probably be `a list defining the super classes`.